### DISCUSSION TITLE
Functionality like loractl in ComfyUI?

### DISCUSSION THREAD

**Discussion by kft334** [comfyanonymous/ComfyUI#2632] [General]
How can I mirror the behavior of this A1111 extension in ComfyUI? https://github.com/cheald/sd-webui-loractl 

I haven't found anything in the default nodes, my installed extensions or any of the available extensions to do this. I rely on loractl quite a bit when using LoRAs in A1111 but I'm trying to migrate to ComfyUI. Does this functionality currently exist in ComfyUI? And is there also a way to do this with ControlNet strength and strengths in general in ComfyUI?

**Comment by brendanhoar** [comment]
IIRC, discussion about this elsewhere indicated that developers were waiting for resolution of https://github.com/comfyanonymous/ComfyUI/pull/2666 , the execution model inversion changes, before attempting work like this.

Anyone want to take a look at to see if this is more feasible now?
